File: src/misc/split_str2float.m
function result = split_str2float(input, header, splitter)
    % SPLIT_STR2INT(input, header, splitter)
    % Splits an input string into Float64 array removing header (if true) and splitting at splitter character.

    s = split(input, splitter);
    s = string(s);
    if header
        s = s(2:end);
    end
    result = zeros(length(s), 1);
    for n = 1:length(s)
        result(n) = str2double(s(n))
    end
end

File: src/misc/Julia-functions/split_str2float.jl
"""
    split_str2float(input::String, header::Bool=true, splitter::Char=' ')

Splits an `input` string into Float64 array removing `header` (if true) and splitting at `splitter` character.
"""

function split_str2float(input::String, header::Bool = true, splitter::Char = ' ')
    s = split(input, splitter)
    if header
        s = s[2:end]
    end
    a = zeros(length(s))
    for n in 1:length(s)
        a[n] = parse(Float64, s[n])
    end
    return a
end
